pytorch_transformers/modeling_roberta.py

```diff
@@ -357,3 +357,142 @@ def forward(self, features, **kwargs):
+
+        logits = self.classifier(sequence_output)
+
+        outputs = (logits,) + outputs[2:]  # add hidden states and attention if they are here
+        if labels is not None:
+            loss_fct = CrossEntropyLoss()
+            # Only keep active parts of the loss
+            if attention_mask is not None:
+                # active_loss = attention_mask.view(-1) == 1
+                active_loss = labels.view(-1) != -1
+                active_logits = logits.view(-1, self.num_labels)[active_loss]
+                active_labels = labels.view(-1)[active_loss]
+                loss = loss_fct(active_logits, active_labels)
+            else:
+                loss = loss_fct(logits.view(-1, self.num_labels), labels.view(-1))
+            outputs = (loss,) + outputs
+
+        return outputs  # (loss), scores, (hidden_states), (attentions)
+
+
+@add_start_docstrings("""RoBERTa Model transformer with a multi-label classification head on top (a linear layer on top of
+    the pooled output) e.g. for BLUE - document multilabel classification tasks. """,
+                      ROBERTA_START_DOCSTRING, ROBERTA_INPUTS_DOCSTRING)
+class RobertaForMultiLabelClassification(BertPreTrainedModel):
+    r"""
+        **labels**: (`optional`) ``torch.FloatTensor`` of shape ``(batch_size,config.num_labels,)``:
+            Labels for computing the multi-label classification.
+            Labels are the arrays of n elements in ``[0, 1]``, corresponding to n = ``config.num_labels`` labels.
+            If the input is labeled l and m (l,m in [0,n-1], the l-th and m-th elements will be 1, otherwise elements will be 0.
+            If the labels are all zero, the input does not have any label.
+
+    Outputs: `Tuple` comprising various elements depending on the configuration (config) and inputs:
+        **loss**: (`optional`, returned when ``labels`` is provided) ``torch.FloatTensor`` of shape ``(1,)``:
+            Classification (or regression if config.num_labels==1) loss.
+        **logits**: ``torch.FloatTensor`` of shape ``(batch_size, config.num_labels)``
+            Classification (or regression if config.num_labels==1) scores (before SoftMax).
+        **hidden_states**: (`optional`, returned when ``config.output_hidden_states=True``)
+            list of ``torch.FloatTensor`` (one for the output of each layer + the output of the embeddings)
+            of shape ``(batch_size, sequence_length, hidden_size)``:
+            Hidden-states of the model at the output of each layer plus the initial embedding outputs.
+        **attentions**: (`optional`, returned when ``config.output_attentions=True``)
+            list of ``torch.FloatTensor`` (one for each layer) of shape ``(batch_size, num_heads, sequence_length, sequence_length)``:
+            Attentions weights after the attention softmax, used to compute the weighted average in the self-attention heads.
+
+    Examples::
+
+        tokenizer = RoertaTokenizer.from_pretrained('roberta-base')
```

Comment:
Similar to the other highlight, his should probably be renamed to `RobertaTokenizer` (typo).